src/controllers/playback/video/index.js

```diff
@@ -1176,6 +1176,20 @@ export default function (view) {
         }
     }
 
+    function autoSelectSubtitleTrack() {
+        const player = currentPlayer;
+        const streams = playbackManager.subtitleTracks(player) ?? [];
+        const currentIndex = playbackManager.getSubtitleStreamIndex(player) ?? -1;
```

Comment:
```suggestion
        const currentIndex = playbackManager.getSubtitleStreamIndex(player);
```

Reply:
i found that `getSubtitleStreamIndex` can be `undefined` on first load with no subtitle selected

Reply:
i could change the `if` statement to include undefined if the nullish coalescence generates excess code

Reply:
> i found that `getSubtitleStreamIndex` can be `undefined` on first load with no subtitle selected

Hmm, you're right. This does sound like a bug. Looks like it's coming from here: https://github.com/jellyfin/jellyfin-web/blob/master/src/components/playback/playbackmanager.js#L2624

You can probably just leave it as-is for now. `getSubtitleStreamIndex` can be null, like for e.g, when playing a trailer, but it wouldn't matter in your case since it'll lead to a no-op.

Reply:
@viown Sounds good. Im not familiar with the PR culture here, do you want me to `Resolve conversation`?